pkg/flare/archive.go

```diff
@@ -42,6 +43,18 @@ const (
 var (
 	pprofURL = fmt.Sprintf("http://127.0.0.1:%s/debug/pprof/goroutine?debug=2",
 		config.Datadog.GetString("expvar_port"))
+
+	// Match other services api keys
+	// It is a best effort to match the api key field without matching our
+	// own already redacted (we don't want to match: **************************abcde)
+	// Basically we allow many special chars while forbidding *
+	otherAPIKeysRx       = regexp.MustCompile(`api_key: [a-zA-Z0-9/\]\[\(\)\{\}><#@$+=]+`)
```

Comment:
nit: forward slash `/` is typically a delimiter, I would maybe escape that even though it's within a character class. Also, we might want to account for blank spaces at the end, maybe we should use a word boundary?
Not sure this expression is exactly what we're looking for, but might be worth testing, you have a great test, so if it doesn't you'll know. Also, maybe account for a variable number of spaces between `api_key:` and the actual key.

```regex
api_key:\s*[a-zA-Z0-9\/\]\[\(\)\{\}><#@$+=]+\b
```

Reply:
Totally legit for the `/` and the white spaces between the `api_key:` and the value, thanks 👍 

I think I can't use the word boundaries because the end of the password could be either a word char (a-Z 0-9) or a special char, so either `\b` or `\B`. I may do `(\b|\B)` but I guess it's like "matching everything", please correct me if you think I'm wrong or if I miss something!

Here's the one I've just pushed:
```
api_key:\s*[a-zA-Z0-9\/\]\[\(\)\{\}><#@$+=]+
```

Reply:
You're right, was overthinking it 👍